plugin/storage/cassandra/Dockerfile

```diff
@@ -1,5 +1,4 @@
-# TODO: replace this by the final Cassandra image
-FROM jpkroehling/cassandra
+FROM cassandra:3.11
```

Comment:
@yurishkuro is this version fine? Shouldn't we use the same version as xdock (3.9) ?